pkgs/development/python-modules/measurement/default.nix

```diff
@@ -1,15 +1,26 @@
-{ lib, fetchPypi, buildPythonPackage, pbr, six, sympy }:
+{ lib, fetchFromGitHub, buildPythonPackage, isPy3k
+, sympy, pytest, pytestrunner, sphinx, setuptools_scm }:
 
 buildPythonPackage rec {
   pname = "measurement";
   version = "3.2.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "352b20f7f0e553236af7c5ed48d091a51cf26061c1a063f46b31706ff7c0d57a";
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "coddingtonbear";
+    repo = "python-measurement";
+    rev = version;
+    sha256 = "1mk9qg1q4cnnipr6xa72i17qvwwhz2hd8p4vlsa9gdzrcv4vr8h9";
   };
 
-  propagatedBuildInputs = [ pbr six sympy ];
+  postPatch = ''
+    sed -i 's|use_scm_version=True|version="${version}"|' setup.py
```

Comment:
so we can drop setuptools_scm

Reply:
Only if we switch from the `fetchFromGitHub` release tarball to `fetchgit` with `leaveDotGit = true`, I think.

```
LookupError: setuptools-scm was unable to detect version for '/build/source'.

Make sure you're either building from a fully intact git repository or PyPI tarballs. Most other sources (such as GitHub's tarballs, a git checkout without the .git folder) don't contain the necessary metadata and will not work.
```

Reply:
it's still in setup_requires, i think it's fine